examples/tutorial/x/blog/handlers.go

```diff
@@ -0,0 +1,88 @@
+package blog
+
+import (
+	"github.com/iov-one/weave"
+	"github.com/iov-one/weave/errors"
+	"github.com/iov-one/weave/orm"
+	"github.com/iov-one/weave/x"
+)
+
+const costPer1000Chars = 1
+
+// RegisterRoutes will instantiate and register
+// all handlers in this package
+func RegisterRoutes(r weave.Registry, auth x.Authenticator) {
+	r.Handle(PathCreatePostMsg, CreatePostMsgHandler{auth, NewPostBucket(), NewBlogBucket()})
```

Comment:
okay, start with first message type...
but maybe you need the `CreateBlog` one first, as you cannot even test this until there is a blog, right?